v1/factories.go

```diff
@@ -12,12 +12,16 @@ import (
 )
 
 // BrokerFactory creates a new object of brokers.Interface
-// Currently only AMQP broker is supported
+// Currently only AMQP/S broker is supported
 func BrokerFactory(cnf *config.Config) (brokers.Interface, error) {
 	if strings.HasPrefix(cnf.Broker, "amqp://") {
 		return brokers.NewAMQPBroker(cnf), nil
 	}
 
+	if strings.HasPrefix(cnf.ResultBackend, "amqps://") {
+		return backends.NewAMQPBackend(cnf), nil
```

Comment:
This line should be:

```go
return brokers.NewAMQPBroker(cnf), nil
```

That's why CI was failing. I have merged the PR, fixed this and pushed to master.